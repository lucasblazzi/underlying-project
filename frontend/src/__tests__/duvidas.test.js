import React from 'react';
import UserProfile from '../pages/duvidas';
import renderer from 'react-test-renderer';

it('PÃ¡gina', () => {
    const tree = renderer.create(<div className="page-content"/>).toJSON();
    expect(tree).toMatchSnapshot();
})