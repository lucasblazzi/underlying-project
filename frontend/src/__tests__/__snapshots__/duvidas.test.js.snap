// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PÃ¡gina 1`] = `
<div
  className="page-content"
/>
`;
